tests/resources/ExpressionCollection.nestml

```diff
@@ -111,26 +203,25 @@ neuron ExpressionCollection:
         test64 = convolve(g_in, spikeInh) * ( V_bounded - E_in )
         test65 = ( -g_L*( V_bounded - E_L ) + I_spike - I_syn_exc - I_syn_inh - w + I_e + I_stim ) / C_m
         test66 = (a*(V_bounded - E_L) - w)/tau_w
-        test67 += spikeExc * nS
-        test68 += spikeInh * nS
+
         #hh_cond_exp_traub_neuron
-        test69 mV = E_L #  Membrane potential
-        test70 = 0nS # Inhibitory synaptic conductance
-        test71 1/ms = 0.032/(ms* mV ) * ( 15. mV - V_m) / ( exp( ( 15. mV - V_m) / 5. mV ) - 1. )
+        test69 = E_L #  Membrane potential
+        test71 = 0.032/(ms* mV ) * ( 15. mV - V_m) / ( exp( ( 15. mV - V_m) / 5. mV ) - 1. )
         inline test72 1/ms = 0.5 /ms * exp( ( 10. mV - V_m ) / 40. mV )
         inline test73 1/ms = 0.32/(ms* mV ) * ( 13. mV - V_m) / ( exp( ( 13. mV - V_m) / 4. mV ) - 1. )
         inline test74 1/ms = 0.28/(ms* mV ) * ( V_m  - 40. mV ) / ( exp( ( V_m - 40. mV ) / 5. mV ) - 1. )
         inline test75 1/ms = 0.128/ms * exp( ( 17. mV - V_m) / 18. mV )
         inline test76 1/ms = ( 4. / ( 1. + exp( ( 40. mV - V_m ) / 5. mV) ) ) / ms
-        test77 real =  alpha_m_init / ( alpha_m_init + beta_m_init )
```

Comment:
Could these three lines be moved to the update block instead in their original form? They are an example of defining a temporary, local variable.